### PR TITLE
Add initial "service" docs

### PR DESCRIPTION
**Pull request by thaJeztah**
/cc @sfsmithcha PTAL

### CODE REVIEW

File: docs/reference/commandline/service_inspect.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+Inspect a service
+
+Options:
+  -f, --format string   Format the output using the given go template
+      --help            Print usage
+  -p, --pretty          Print the information in a human friendly format.
+```
+
+
+Inspects the specified service. This command has to be run targeting a Manager
```
Comment (sfsmithcha) [inline docs/reference/commandline/service_inspect.md]: manager node  no caps

File: docs/reference/commandline/service_create.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      --restart-window value         Window used to evalulate the restart policy (default none)
+      --stop-grace-period value      Time to wait before force killing a container (default none)
+      --update-delay duration        Delay between updates
+      --update-parallelism uint      Maximum number of tasks updated simultaneously
+  -u, --user string                  Username or UID
+  -w, --workdir string               Working directory inside the container
+```
+
+Creates a service as described by the specified parameters. This command has to
+be run targeting a Manager node.
```
Comment (sfsmithcha) [inline docs/reference/commandline/service_create.md]: "manager node" no caps

File: docs/reference/commandline/service_create.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  redis:3.0.6
+```
+
+For more information about labels, refer to [apply custom
+metadata](../../userguide/labels-custom-metadata.md)
+
+### Service mode
+
+Is this a replicated service or a global service. A replicated service runs as
+many tasks as specified, while a global service runs on each node in the swarm.
```
Comment (sfsmithcha) [inline docs/reference/commandline/service_create.md]: global service runs on each "active" node in the cluster. (for ex. not on "drain" nodes).

File: docs/reference/commandline/service_inspect.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+### Inspecting a service  by name or ID
+
+You can inspect a service, either by its *name*, or *ID*
+
+For example, given the following services, `docker service inspect frontend`,
+and `docker service inspect `c8wgl7q4ndfd` produce the same result:
+
+```bash
+ID            NAME      REPLICAS  IMAGE         COMMAND
```
Comment (sfsmithcha) [inline docs/reference/commandline/service_inspect.md]: this looks like output of `docker service ls`:
```
$ docker service ls
ID            NAME        SCALE  IMAGE   COMMAND
7wwt8q37h6um  helloworld  1/1      alpine  ping docker.com

$ docker service inspect helloworld --pretty
ID:		7wwt8q37h6um2djxglngghhuq
Name:		helloworld
Mode:		REPLICATED
 Replicas:		1
Placement:
 Strategy:	SPREAD
UpateConfig:
 Parallelism:	1
ContainerSpec:
 Image:		alpine
 Args:	ping docker.com
```

File: docs/reference/commandline/service_inspect.md (hunk 1 above)
Comment (thaJeztah) [inline docs/reference/commandline/service_inspect.md]: thanks, copied that from the original content, so thought it was meant to be like this; I'll update

File: docs/reference/commandline/service_inspect.md (hunk 4 above)
Comment (thaJeztah) [inline docs/reference/commandline/service_inspect.md]: ah, yes, I can add that; wanted to show which services are there

File: docs/reference/commandline/service_tasks.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+### Listing the tasks that are part of a service
+
+The following command shows all the tasks that are part of the `redis` service:
+
+```bash
+$ docker service tasks redis
+ID                         NAME      SERVICE IMAGE        LAST STATE          DESIRED STATE  NODE
+0qihejybwf1x5vqi8lgzlgnpq  redis.1   redis   redis:3.0.6  RUNNING 8 seconds   RUNNING        manager1
+bk658fpbex0d57cqcwoe3jthu  redis.2   redis   redis:3.0.6  RUNNING 9 seconds   RUNNING        worker2
+5ls5s5fldaqg37s9pwayjecrf  redis.3   redis   redis:3.0.6  RUNNING 9 seconds   RUNNING        worker1
```
Comment (sfsmithcha) [inline docs/reference/commandline/service_tasks.md]: Not sure why `LAST STATE` / `DESIRED STATE` show in all caps for you. This is supposed to be "title" case:
```
docker service tasks helloworld
ID                         NAME          SERVICE     IMAGE   LAST STATE       DESIRED STATE  NODE
0unc3t7qtto2ras4l2voju2nt  helloworld.1  helloworld  alpine  Running 3 hours  Running        manager1
```

**Comment by sfsmithcha** [comment]
other than comments LGTM. Please see first instance of `docker service inspect` looks like output of `docker service ls`

**Comment by sfsmithcha** [comment]
LGTM

**Comment by thaJeztah** [comment]
thanks!

**Comment by thaJeztah** [comment]
CI fail is only the expected issues;

```
# Filtered (engine) Summary:

* link error: (in page engine/installation/mac.md) /docker-for-mac/index.md
* link error: (in page engine/getstarted/last_page.md) /docker-for-mac/index.md
* link error: (in page engine/getstarted/step_one.md) /docker-for-mac/index.md
* link error: (in page engine/getstarted/step_one.md) /docker-for-windows/index.md
* link error: (in page engine/installation/windows.md) /docker-for-windows/index.md
* link error: (in page engine/getstarted/last_page.md) /docker-for-windows/index.md
```

**Pull request merged**